Demo/Editor/DemoSceneLoader.cs

```diff
@@ -30,6 +32,22 @@ public static void LoadDemoScene()
 
             EditorSceneManager.SaveCurrentModifiedScenesIfUserWantsTo();
             EditorSceneManager.OpenScene(demoScenePath);
+
+#if VR_BUILDER && VR_BUILDER_XR_INTERACTION
```

Comment:
Can you explain me what's the differente between VR_BUILDER and VR_BUILDER_XR_INTERACTION?

Reply:
VR_BUILDER is the core package.
VR_BUILDER_XR_INTERACTION is the XR Interaction component, i.e. the XRI-specific stuff, which is still part of core but can be enabled or disabled in the settings. For example, if you wanted to use VRIF you would have no need for XRI and the XR Interaction component, so you could remove both from your project.

Here the VR_BUILDER define might be unnecessary but better safe than sorry.